frontend/src/store/reducers/reducers.js

```diff
@@ -0,0 +1,19 @@
+import * as actionTypes from "../actionTypes";
+
+const initialState = { categories: [] };
+
+const rootReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.CATEGORIES:
+      return {
+        ...state,
+        categories: action.categories
+      };
+    default:
+      return {
+        ...state
+      };
```

Comment:
always live empty line before default
```suggestion
      };
      
    default:
      return {
        ...state
      };
```